popup.js

```diff
@@ -0,0 +1,156 @@
+
+let currentVid;
+let current = [];
+
+// adding a new bookmark row to the popup
+const addNewBm = (bms, bookmark) => {
+
+	// bookmark title element
+	const bmTitle = document.createElement('div');
+	bmTitle.textContent = bookmark.desc;
+	bmTitle.className = 'bookmark-title';
+
+	const editBmTitle = document.createElement('img');
+	editBmTitle.src = 'assets/edit.png';
+	editBmTitle.title = 'Edit bookmark Title';
+	editBmTitle.addEventListener('click', onEdit);
+
+	const playBm = document.createElement('img');
+	playBm.src = 'assets/play.png';
+	playBm.title = 'Start play from bookmark';
+	playBm.addEventListener('click', onPlay);
+
+	const deleteBm = document.createElement('img');
+	deleteBm.src = 'assets/delete.png';
+	deleteBm.title = 'Delete bookmark';
+	deleteBm.addEventListener('click', onDelete);
+	
+	// bookmark controls - edit bookmark title, play from bookmark
+	const bmControls = document.createElement('div');
+	bmControls.className = 'bookmark-controls';
```

Comment:
alot of duplicate code here, code reduce a lot of code but using an object and iterating over it but this is okay for now

Reply:
agreed. can probably create a class to abstract tons of this away but fine for right now. I'll put up a follow-up pr to clean this.

Reply:
Sure, I'll add these changes in my next pr